### PR TITLE
🌐 [i18n-KO] Translated `tasks/image_classification.mdx` to Korean

### PR DESCRIPTION
**Pull request by 0525hhgus**
# What does this PR do?

Translated the `tasks/image_classification.mdx` file of the documentation to Korean.
Thank you in advance for your review 😄 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? 
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR?
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/tasks/image_classification.mdx [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+>>> food = load_dataset("food101", split="train[:5000]")
+```
+
+데이터 세트의 `train`을 [`~datasets.Dataset.train_test_split`] 메소드를 사용하여 훈련 및 테스트 세트로 분할하세요:
+
+```py
+>>> food = food.train_test_split(test_size=0.2)
+```
+
+그런 다음 예시를 살펴보세요:
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/image_classification.mdx:69]: 그런 다음
>> 그리고
가 어떨까합니다

File: docs/source/ko/tasks/image_classification.mdx [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+...     return examples
+```
+
+전체 데이터 세트에 전처리 기능을 적용하려면 🤗 Datasets [`~datasets.Dataset.with_transform`]을 사용합니다. 데이터 세트의 요소를 가져올 때 변환이 즉시 적용됩니다:
+
+```py
+>>> food = food.with_transform(transforms)
+```
+
+이제 [`DefaultDataCollator`]를 사용하여 예제 배치를 만듭니다. 🤗 Transformers의 다른 데이터 수집기와 달리, `DefaultDataCollator`는 패딩과 같은 추가적인 전처리를 적용하지 않습니다.
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/image_classification.mdx:144]: 데이터 수집기
> 데이터 콜레이터
인지 확인이 필요할 것 같아요

File: docs/source/ko/tasks/image_classification.mdx [hunk 3]
```diff
@@ -0,0 +1,542 @@
...
+
+```py
+>>> from transformers import AutoImageProcessor
+>>> import torch
+
+>>> image_processor = AutoImageProcessor.from_pretrained("my_awesome_food_model")
+>>> inputs = image_processor(image, return_tensors="pt")
+```
+
+입력을 모델에 전달하고 logits을 반환합니다:
```
Comment (gabrielwithappy) [inline docs/source/ko/tasks/image_classification.mdx:493]: 영어 원문에서 `logits`로 사용했는지 확인이 필요할 것 같아요

File: docs/source/ko/tasks/image_classification.mdx [hunk 4]
```diff
@@ -0,0 +1,542 @@
...
+
+# 이미지 분류[[image-classification]]
+
+[[open-in-colab]]
+
+<Youtube id="tjAIM7BOYhw"/>
+
+이미지 분류는 이미지에 레이블 또는 클래스를 할당합니다. 텍스트 또는 오디오 분류와 달리 입력은
+이미지를 구성하는 픽셀 값입니다. 자연재해 후 피해 감지, 농작물 건강 모니터링, 의료 이미지에서 질병의 징후 검사 지원 등
+이미지 분류에서의 다양한 응용 사례가 있습니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/image_classification.mdx:21]:
```suggestion
이미지를 구성하는 픽셀 값입니다. 이미지 분류에는 자연재해 후 피해 감지, 농작물 건강 모니터링, 의료 이미지에서 질병의 징후 검사 지원 등
다양한 응용 사례가 있습니다.
```
이런 문장은 어떨까요?

File: docs/source/ko/tasks/image_classification.mdx (hunk 2 above)
Comment (sim-so) [reply docs/source/ko/tasks/image_classification.mdx:144]:
```suggestion
이제 [`DefaultDataCollator`]를 사용하여 예제 배치를 만듭니다. 🤗 Transformers의 다른 데이터 콜레이터와 달리, `DefaultDataCollator`는 패딩과 같은 추가적인 전처리를 적용하지 않습니다.
```
원문에서 data collator로 사용된 다른 부분과 번역어를 맞추었습니다.

File: docs/source/ko/tasks/image_classification.mdx (hunk 3 above)
Comment (sim-so) [reply docs/source/ko/tasks/image_classification.mdx:493]: 다른 task 문서에서는 `logit`으로 표기한 것과 달리, 여기서는 원문에서도 backtick 없이 logit으로 썼네요 🤔

**Review by sim-so — CHANGES_REQUESTED** [review]
고생 많으셨습니다! 😃 
많지 않지만 몇 가지 코멘트를 남겼어요

File: docs/source/ko/tasks/image_classification.mdx (hunk 4 above)
Comment (0525hhgus) [reply docs/source/ko/tasks/image_classification.mdx:21]: 더 자연스러운 문장이군요 👍 반영했습니다!

File: docs/source/ko/tasks/image_classification.mdx (hunk 3 above)
Comment (0525hhgus) [reply docs/source/ko/tasks/image_classification.mdx:493]: 다른 문서는 backtick이 들어가 있는데, 이 문서만 없군요 😮 원문을 따라서 logits로 작성해놓겠습니다..!

File: docs/source/ko/tasks/image_classification.mdx (hunk 1 above)
Comment (0525hhgus) [reply docs/source/ko/tasks/image_classification.mdx:69]: 더 자연스러운 표현 감사합니다. 반영했습니다!

**Comment by 0525hhgus** [comment]
May you please review this PR? 😄 
@sgugger, @ArthurZucker, @eunseojo

**Review by sgugger — APPROVED** [review]
Thanks!
